specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/common.json

```diff
@@ -5,6 +5,7 @@
     "description": "The key vault client performs cryptographic key operations and vault operations against the Key Vault service.",
     "version": "7.4-preview.1"
   },
+  "x-ms-parameter-location": "client",
```

Comment:
This needs to go on the ApiVersionParameter itself, not just arbitrarily in the swagger. See https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-parameter-location